testing/configitemtest.cc

```diff
@@ -0,0 +1,285 @@
+#include <gtest/gtest.h>
+#include <gmock/gmock.h>
+
+#include "game/configitem.hh"
+
+using ::testing::ElementsAre;
+using ::testing::IsEmpty;
+
+namespace {
```

Comment:
not sure this namespace is really useful

Reply:
It was suggested by google. Don't know when it is a must.